extensions/positron-r/src/util.ts

```diff
@@ -42,8 +43,13 @@ export function timeout(ms: number, reason: string) {
 }
 
 export function readLines(pth: string): Array<string> {
-	const bigString = fs.readFileSync(pth, 'utf8');
-	return bigString.split(/\r?\n/);
+	try {
```

Comment:
In this journey, I went through a state where I mistakenly read from a non-existent file and discovered we needed better error handling here.